ftplugin/orgmode/plugins/Export.py

```diff
@@ -81,7 +81,7 @@ def _export(cls, format_):
 		p.wait()
 
 		if p.returncode != 0 or settings.get(u'org_export_verbose') == 1:
-			echom('\n'.join(p.communicate()))
+			echom('\n'.join(map(lambda x: x.decode(), p.communicate())))
```

Comment:
for consistent decoding use ``u_decode`` function from here [file](https://github.com/jceb/vim-orgmode/blob/master/ftplugin/orgmode/py3compat/encode_compatibility.py)